packages/docs/docs/updating.mdx

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_position: 100
+slug: /updating
+---
+
+# Update Guide
+
+The latest patch notes can be found
+[here](https://github.com/motion-canvas/motion-canvas/releases).
+
+To update all the `@motion-canvas` packages to the latest version use the
+following:
+
+```bash
+npm update --save @motion-canvas/2d @motion-canvas/core @motion-canvas/ui @motion-canvas/vite-plugin
+```
+
+## Toubleshooting
+
+If you get any errors try to remove the `package-lock.json` and the
+`node_modules` folder.  
+Then run `npm install` again.
```

Comment:
nit: This probably should be on the previous line since it's not double spaced (and therefore should render on the same line as above)

Reply:
Should be fixed with the one-liner you proposed :)